Splitio.Redis/Services/Cache/Classes/RedisSplitCacheAsync.cs

```diff
@@ -0,0 +1,155 @@
+﻿using Newtonsoft.Json;
+using Splitio.Domain;
+using Splitio.Redis.Services.Cache.Interfaces;
+using Splitio.Services.Cache.Interfaces;
+using Splitio.Services.Parsing.Interfaces;
+using StackExchange.Redis;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Splitio.Redis.Services.Cache.Classes
+{
+    public class RedisSplitCacheAsync : RedisCacheBase, ISplitCacheAsync
```

Comment:
if the class is already named as `Async` maybe we should not add `Async` over each method

Reply:
nop, the `Async` suffix is part of good practice of the implementation.